components/posts/posts.php

```diff
@@ -781,25 +781,29 @@ function mm_vc_posts() {
 		return;
 	}
 
+	$titles         = mm_get_post_titles_for_vc( 'mm-posts' );
 	$post_types     = mm_get_post_types_for_vc( 'mm-posts' );
 	$taxonomies     = mm_get_taxonomies_for_vc( 'mm-posts' );
 	$heading_levels = mm_get_heading_levels_for_vc( 'mm-posts' );
 	$image_sizes    = mm_get_image_sizes_for_vc( 'mm-posts' );
 	$templates      = mm_get_mm_posts_templates_for_vc( 'mm-posts' );
 
 	vc_map( array(
-		'name'     => __( 'Posts', 'mm-components' ),
-		'base'     => 'mm_posts',
-		'class'    => '',
-		'icon'     => MM_COMPONENTS_ASSETS_URL . 'component-icon.png',
-		'category' => __( 'Content', 'mm-components' ),
-		'params'   => array(
+		'name'              => __( 'Posts', 'mm-components' ),
+		'base'              => 'mm_posts',
+		'class'             => '',
+		'icon'              => MM_COMPONENTS_ASSETS_URL . 'component-icon.png',
+		'category'          => __( 'Content', 'mm-components' ),
+		'admin_enqueue_css' => MM_COMPONENTS_URL . '/css/post-titles-autocomplete.css',
+		'params'            => array(
 			array(
-				'type'        => 'textfield',
-				'heading'     => __( 'Post IDs', 'mm-components' ),
-				'param_name'  => 'post_ids',
-				'description' => __( 'Enter post IDs to display (separated by commas)', 'mm-components' ),
-				'value'       => '',
+				'type'        => 'autocomplete',
+				'heading'     => __( 'Post/Page Titles', 'mm-components' ),
+				'param_name'  => 'post_titles',
+				'description' => __( 'Enter specific post or page titles to display', 'mm-components' ),
```

Comment:
Bonus points if you make this dynamic based on the current post type. You could %s where it says `post or page` and `sprintf` to get the variable into the translation, then set the variable to the post type for the current post. The `global $post` object will give you the post ID and post type, although extra bonus points if you reach into the labels on the post type object and extract a proper Single label for the post type.

Using sprintf() with translation looks like this:

```
$topic = 'stuff';
$var = sprintf( __( 'Some string about %s', 'xxx' ), $topic );
```